Assets/Scripts/Utilities/DayCycleManager.cs

```diff
@@ -61,34 +67,64 @@ public static void RegisterEndOfDayCallback( EndOfDayCallback callback )
 
 	public static void DeregisterEndOfDayCallback( EndOfDayCallback callback )
 	{
-		instance._endOfDayCallback += callback;
+		instance._endOfDayCallback -= callback;
 	}
 
 	void StartMidnightOverlay()
 	{
-		_hasStartedMidnightOverlay = true;
 		StartCoroutine( FadeInMidnightOverlay() );
 	}
 
-	void EndMidnightOverlay()
+	void StartMorningOverlay()
 	{
-		_hasStartedMidnightOverlay = false;
-
-		StopCoroutine( _midnightOverlayCoroutine );
-
-		Color overlayColor = _midnightOverlay.color;
-		overlayColor.a = 0.0f;
-		_midnightOverlay.color = overlayColor;
+		StartCoroutine( FadeOutMidnightOverlay() );
```

Comment:
Are StartMidnightOverlay() and StartMorningOverlay() worth having? They both only have one line each. I think it'd be clearer to remove them